SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -50,6 +52,9 @@
 from Configuration.StandardSequences.Eras import eras
 if eras.fastSim.isChosen():
     hcalSimBlock.hitsProducer = cms.string('famosSimHits')
+if eras.run2_HF_2016.isChosen():
```

Comment:
I leave the final word to @Dr15Jones, but actually `toModify()` applies the changes immediately (at least currently, and I've understood from Chris that this effect would be preserved in the future too).